warehouse/locale/id/LC_MESSAGES/messages.po

```diff
@@ -11,21 +11,34 @@
 # Habib Rohman <revolusi147id@gmail.com>, 2020, 2021.
 # Syahmin Sukhairi <syahmin@gmail.com>, 2021.
 # Erikson <erikson99.id@gmail.com>, 2021.
+<<<<<<< HEAD
+=======
+# Jacque Fresco <aidter@use.startmail.com>, 2021.
+>>>>>>> 78cbf723 (Translations update from Weblate)
```

Comment:
@di bad merge?

Reply:
Fixed in https://github.com/pypa/warehouse/pull/9481